packages/flutter_tools/lib/src/test/coverage_collector.dart

```diff
@@ -42,6 +42,8 @@ class CoverageCollector extends TestWatcher {
 
     final int pid = process.pid;
     int exitCode;
+    // Callback hookups' Futures are uninteresting.
+    // ignore: unawaited_futures
```

Comment:
this comment doesn't accurately describe why we're not awaiting this future. In this case it's a combination of the specific closure not risking throwing anything, and the caller being the one in charge of the Process' lifetime.